Appraisals

```diff
@@ -1,4 +1,4 @@
-%w(5.0.7 5.1.7 5.2.3 6.0.0).each do |ar_version|
+%w(5.2.3 6.0.0).each do |ar_version|
```

Comment:
do we want to set the minimum version in the gemspec

does this mean we want to delete gemfiles as well?